partner_socialmedia/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Humanitarian Logistics Organisation e.V. - Stefan Becker
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Socialmedia',
+    'summary': 'Add social media fields to contacts',
+    'version': '10.0.1.0.0',
+    'author': "humanilog",
```

Comment:
@funbaker IMO author should like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)